src/components/Description/Description.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import { FiArrowUpRight } from 'react-icons/fi';
+import { Description } from './Description';
+
+export default {
+  title: 'Components/Description',
+};
+
+const args = {
+  preIcon: <FiArrowUpRight />,
+  title: 'Website name',
+  website: 'example.massa',
+  description: 'Website’s description up to 120 characters',
+  numberVote: 4,
+  voted: 1 as 0 | 1 | -1,
+  onClickVotes: (arg: number) => console.log(arg),
+};
+
+export const _Primary = {
+  render: () => <Description {...args} />,
```

Comment:
In the description I can see the large version and the small version. Are they this primary or secondary!?

If yes, we must change because primary/secondary is a variant in colors, not in size. If we want have variants in size let's use the same standard as in `<Balance>` component. We use  `size: 'xs' | 'lg';`

Have a look there and let me know if you need help

Reply:
done